tests/gflownet/envs/test_tree.py

```diff
@@ -573,6 +573,11 @@ def test__get_masks_backward__returns_expected(
     assert tree_d3.get_mask_invalid_actions_backward() == mask
 
 
+def test__all_env_common__standard__treedd3(tree_d3):
```

Comment:
```suggestion
def test__all_env_common__standard__tree_d3(tree_d3):
```
Was that intended or a typo?